pkg/rancher/migrations.go

```diff
@@ -135,6 +147,43 @@ func createOrUpdateConfigMap(configMapClient controllerv1.ConfigMapClient, cm *v
 	return err
 }
 
+func versionTombstone(configMapController controllerv1.ConfigMapController) error {
+	cm, err := getConfigMap(configMapController, rancherVersionTombstoneConfig)
+	if err != nil || cm == nil {
+		return err
+	}
```

Comment:
Please let me know if I missed anything:  
It seems we introduce a new ComfigMap here, since this ConfigMap has never existed in previous Rancher setups, how would this check pass? Will it prevent a fresh-install Rancher from bootstrapping?